# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome14/gtk-perl10/gendefs.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./tools/gendefs.pl~	Tue Mar  9 19:46:14 1999
+++ ./tools/gendefs.pl	Tue Apr 27 16:08:13 1999
@@ -517,6 +517,10 @@
 #undef _
 #endif
 
+#ifdef dirty
+#undef dirty
+#endif
+
 #ifndef Perl${FilePrefix}DeclareFunc
 #include "Perl${FilePrefix}Int.h"
 #endif
